etl/steps/data/garden/war/2023-09-21/cow.py

```diff
@@ -1223,6 +1223,7 @@ def estimate_metrics_locations(tb_chupilkin: Table, tb_system: Table, tb_partici
     tb_locations_inter["is_location_of_conflict"] = tb_locations_inter["is_location_of_conflict"].fillna(0)
     # Filter irrelevant entries
     tb_locations_inter = tb_locations_inter[tb_locations_inter["year"] <= tb_chupilkin["year"].max()]
+    tb_locations_inter = tb_locations_inter.astype({"country": str})
```

Comment:
I think a better solution is to use `.astype("string")`.

is funny how this is my own step, and i'm suggesting how to improve your improvement on my code.